common/djangoapps/third_party_auth/signals/tests/test_handlers.py

```diff
@@ -33,100 +33,77 @@ def setUp(self):
             entity_id='https://existing.example.com'
         )
 
-    @ddt.data(
-        # Case 1: Tests behavior when SAML config signal handlers are disabled
-        # Verifies that basic attributes are set but no provider updates are attempted
-        {
-            'enabled': False,
-            'simulate_error': False,
-            'description': 'handlers disabled',
-            'expected_calls': [
+    @mock.patch('common.djangoapps.third_party_auth.signals.handlers.set_custom_attribute')
+    def test_saml_config_signal_handlers_disabled(self, mock_set_custom_attribute):
+        """
+        Test behavior when SAML config signal handlers are disabled.
+
+        Verifies that basic attributes are set but no provider updates are attempted.
+        """
+        with override_settings(ENABLE_SAML_CONFIG_SIGNAL_HANDLERS=False):
+            self.saml_config.entity_id = 'https://updated.example.com'
+            self.saml_config.save()
+
+            expected_calls = [
                 call('saml_config_signal.enabled', False),
-                call('saml_config_signal.new_config_id', 'CONFIG_ID'),
+                call('saml_config_signal.new_config_id', self.saml_config.id),
                 call('saml_config_signal.slug', 'test-config'),
-            ],
-            'expected_call_count': 3,
-        },
-        # Case 2: Tests behavior when SAML config signal handlers are enabled
-        # Verifies that attributes are set and provider updates are attempted successfully
-        {
-            'enabled': True,
-            'simulate_error': False,
-            'description': 'handlers enabled',
-            'expected_calls': [
-                call('saml_config_signal.enabled', True),
```

Comment:
@ktyagiapphelix2u: Sorry if it wasn't clear, but I wanted thes custom attribute assertions for these tests moved to our new ddt test, which is the new test that serves as a replacement for this test.